gateway/app/main/routes.py

```diff
@@ -5,12 +5,14 @@
 from flask import render_template, Response, request
 from . import main
 import json
-
+import random
+import os
 import requests
 
 
 # All grid nodes registered at grid network will be stored here
 grid_nodes = {}
+n_replica = os.getenv("REPLICAS", None)
```

Comment:
can we move this to where the other env variables are declared?

Also, it would be nice to have the option of setting this using an argument.

Reply:
I think in the future this can actually be supported as an argument to the function, but for now, an env variable/script argument is fine.

Reply:
Done! (6ace96c)